hover-provider-plugin/LICENSE

```diff
@@ -0,0 +1,81 @@
+Eclipse Public License - v 2.0
```

Comment:
formatting of the license is stange
compare with :
https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt

Reply:
Thanks, I'll fix it here and in PR for `che-generator-plugin`.